### PR TITLE
gst-devtools 1.18.0 (new formula)

### PR DESCRIPTION
**Pull request by tschoonj**
Replaces gst-validate, which is now a part of gst-devtools

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/gst-devtools.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      system "meson", *args, ".."
+      system "ninja", "-v"
+      system "ninja", "install", "-v"
+    end
+
+    rewrite_shebang detected_python_shebang, bin/"gst-validate-launcher"
+  end
+
+  test do
+    system "#{bin}/gst-validate-launcher", "--usage"
```
Comment (SMillerDev) [inline Formula/gst-devtools.rb:38]: Can we get a better test than this? This is similar to a `--help` test right?

Comment (tschoonj) [reply Formula/gst-devtools.rb:38]: I copied the test block from `gst-validate`... honestly I have no idea what this package does, I have never used GStreamer 😄

Comment (SMillerDev) [reply Formula/gst-devtools.rb:38]: Time to find out 😄

Comment (tschoonj) [reply Formula/gst-devtools.rb:38]: Sorry, but I don't have that kind of time right now. 

Would be grateful if you could merge it in regardless.

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**